Userland/Libraries/LibJS/Runtime/AbstractOperations.cpp

```diff
@@ -235,11 +235,10 @@ bool is_compatible_property_descriptor(bool extensible, PropertyDescriptor const
     return validate_and_apply_property_descriptor(nullptr, "", extensible, descriptor, current);
 }
 
-// 10.1.6.3 ValidateAndApplyPropertyDescriptor ( O, P, extensible, Desc, current ), https://tc39.es/ecma262/#sec-validateandapplypropertydescriptor
+// 10.1.6.3 ValidateAndApplyPropertyDescriptor ( O, P, extensible, Desc, current ), https://262.ecma-international.org/15.0/index.html#sec-validateandapplypropertydescriptor
```

Comment:
Let's not change spec links here. We can discuss on discord, but my general feeling is that:
1. If I see a spec link with the current version, that does not tell me the implementation is up-to-date, because we target the main spec - not the last released spec.
2. If I see a spec link with the previous version, that does not tell me the implementation is out-of-date, because the spec could very well have not changed.

Reply:
My argument for 2. is that it at least tells you that the implementation _could_ be out of date, but I did not consider 1., and that sounds like reason enough to link the Draft spec :+1: .